vllm/v1/engine/core.py

```diff
@@ -317,27 +320,35 @@ def __init__(
         log_stats: bool,
         engine_index: int = 0,
     ):
-        super().__init__(vllm_config, executor_class, log_stats)
+        input_queue: queue.Queue[tuple[EngineCoreRequestType,
+                                       Any]] = queue.Queue()
```

Comment:
```suggestion
        input_queue = queue.Queue[tuple[EngineCoreRequestType, Any]]()
```

No need to repeat the type annotation (since [PEP 560](https://peps.python.org/pep-0560/)). Same for other instances where Queue is initialized,